docs/terminal/basics.md

```diff
@@ -457,6 +457,15 @@ To workaround this you have two options:
 * Ensure that you don't set `$EDITOR` to `vi(m)`. However, this isn't an option if you want your Git editor to work.
 * Add `bindkey -e` to your init script to set Emacs explicitly.
 
+How can I configure a keybinding which maps `Cmd+.` to `Ctrl+C` in order to align with behavior in the macOS terminal?
+
+```json
+// Break: ctrl+C
+registerSendSequenceKeybinding(String.fromCharCode('C'.charCodeAt(0) - 64), {
+	mac: { primary: KeyMod.CtrlCmd | KeyCode.Period }
+});
+```
```

Comment:
The macOS default terminal uses `kbstyle(cmd+.)` to perform the same as `kbstyle(ctrl+c)`. To get this behavior in VS Code, add this [custom keybinding](https://code.visualstudio.com/docs/getstarted/keybindings):

```suggestion
```json
{
	"key": "cmd+.",
	"command": "workbench.action.terminal.sendSequence",
	"when": "terminalFocus",
	"args": { "text": "\u0003" }
}
```
```

Reply:
Opps I did the suggestion wrong, confusing because of multiple ``` blocks. It should say this before the json:

```
The macOS default terminal uses `kbstyle(cmd+.)` to perform the same as `kbstyle(ctrl+c)`. To get this behavior in VS Code, add this [custom keybinding](https://code.visualstudio.com/docs/getstarted/keybindings):
```